src/pages/PlayMode/PlayMode.js

```diff
@@ -42,22 +47,79 @@ export function PlayMode({
       .finally(() => setLoading(false));
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [fetchDeckById, match.params.deckId, touchDeck]);
+
   useEffect(() => {
     if (deckInPlaySession) {
-      touchDeck(deckInPlaySession);
+      touchDeck(deckInPlaySession[0]);
+      setCardRanks(
+        deckInPlaySession.map(card => {
+          return card.value;
+        })
+      );
+      axiosWithAuth().post("/cards/initialise", {
```

Comment:
Not sure on why change it from a deck object to just an array of flashcards, any reason in particular?